src/Block/Service/AbstractAdminBlockService.php

```diff
@@ -32,14 +31,6 @@
  */
 abstract class AbstractAdminBlockService extends AbstractBlockService implements AdminBlockServiceInterface
 {
-    /**
-     * @param string $name
-     */
-    public function __construct($name, EngineInterface $templating)
```

Comment:
The only change I see at first glance is that parent definition is allowing `null` for the 2nd argument.

Reply:
abstract class AbstractAdminBlockService extends AbstractBlockService implements AdminBlockServiceInterface
```php
abstract class AbstractBlockService implements BlockServiceInterface
{
    ...

    /**
     * NEXT_MAJOR: Make `$twig` argument mandatory and remove other arguments.
     *
     * @param Environment|EngineInterface|string $templatingOrDeprecatedName
     */
    public function __construct($templatingOrDeprecatedName = null, EngineInterface $templating = null)
    ...
```

Reply:
Yes, that's exactly what I'm pointing (`EngineInterface $templating = null`). Although I don't know if it worths.

Reply:
This change is require to https://github.com/sonata-project/SonataBlockBundle/issues/700